src/shared/components/axisTicks/TimeTicksOptions.tsx

```diff
@@ -0,0 +1,55 @@
+// Libraries
+import React, {FC} from 'react'
+
+// Components
+import {ValueTickInput} from 'src/shared/components/axisTicks/ValueTickInput'
+import {TimeTickInput} from 'src/shared/components/axisTicks/TimeTickInput'
+import {Grid} from '@influxdata/clockface'
+
+interface TimeTicksOptionsProps {
+  axisName: string
+  tickOptions: string[]
+  totalTicks: number | string
+  tickStart: number | string
+  tickStep: number | string
+  setOptions: (optionName: string, arg: string[] | number) => void
+}
+
+export const TimeTicksOptions: FC<TimeTicksOptionsProps> = props => {
+  const {
+    axisName,
+    tickOptions,
+    totalTicks,
+    tickStart,
+    tickStep,
+    setOptions,
+  } = props
+  return (
+    <>
```

Comment:
```suggestion
```

this isn't needed right? `Grid.Row` contains all the `ValueTickInputs`

Reply:
Correct, not needed. I had changed back and forth between a few designs. Forgot to remove the fragment tag. Done now.